android/SmartPendant/app/src/main/java/ddioriomendes/smartpendant/BluetoothSerial.java

```diff
@@ -54,12 +55,12 @@ public BluetoothSerial(Context context, BluetoothSerialListener externalListener
     }
 
     public void findBondedDevice(String deviceNamePrefix) {
-        asyncFindBondedDevice.execute(deviceNamePrefix);

```

Comment:
Static AsyncTasks now are instantieted every call.
This avoids AsyncTask already started exception.